src/main/java/se/uu/elephas/services/OrderServiceImpl.java

```diff
@@ -138,4 +140,14 @@ public Order proceedOrder(Long idUser) {
 
     }
 
+    public int sizeCart(Long idUser){
+        Order basket=getBasketOfUser(idUser).iterator().next();
+        Iterable<OrderItem> cart =getOrderItemsByOrderId(basket.getIdOrder());
+        int mycounter=0;
+        for (OrderItem item:cart){
+            mycounter = mycounter + 1*item.getQuantity();
```

Comment:
is it neccessary to multiply with 1?

Reply:
haha you are right it was my idea ahahahhah
:1st_place_medal: